Doc/library/decimal.rst

```diff
@@ -374,10 +374,24 @@ Decimal objects
    returns ``Decimal('1.414')``.
 
    If *value* is a :class:`float`, the binary floating point value is losslessly
-   converted to its exact decimal equivalent.  This conversion can often require
-   53 or more digits of precision.  For example, ``Decimal(float('1.1'))``
-   converts to
-   ``Decimal('1.100000000000000088817841970012523233890533447265625')``.
+   converted to its exact decimal equivalent.  This means, for example, that
```

Comment:
I would leave this section alone.  The information in it was correct and helpfully pointed-out a surprising fact of how many digits were required for a lossless conversion.

Also, drop the additional doctests here.  The repeat information that was already in the Quickstart section. This current section on the Decimal constructor is already very long.  If it gets longer, no one will read it.